### PR TITLE
Updates to VPN SaaSboard

### PR DESCRIPTION
**Pull request by wwyc**
* changed retention dashboard visualizations to not include 'not retained'
* added bullets to body text
* replaced subs aggregate tables

### CODE REVIEW

File: mozilla_vpn/explores/subscriptions.explore.lkml [hunk 1]
```diff
@@ -90,10 +87,8 @@ explore: +subscriptions {
       ]
     }
 
-    # Please specify a datagroup_trigger or sql_trigger_value
-    # See https://looker.com/docs/r/lookml/types/aggregate_table/materialization
     materialization: {
-      sql_trigger_value: SELECT CURRENT_DATE();;
+      sql_trigger_value:  v
```
Comment (relud) [inline mozilla_vpn/explores/subscriptions.explore.lkml:91]: this doesn't seem right

File: mozilla_vpn/explores/subscriptions.explore.lkml [hunk 2]
```diff
@@ -73,11 +72,9 @@ explore: +subscriptions {
       ]
     }
 
-    # Please specify a datagroup_trigger or sql_trigger_value
-    # See https://looker.com/docs/r/lookml/types/aggregate_table/materialization
     materialization: {
       sql_trigger_value: SELECT CURRENT_DATE();;
-    }
+      }
```
Comment (relud) [inline mozilla_vpn/explores/subscriptions.explore.lkml:77]: the previous indentation here was more consistent

**Review by relud — APPROVED** [review]
r+wc

File: mozilla_vpn/explores/subscriptions.explore.lkml (hunk 1 above)
Comment (wwyc) [reply mozilla_vpn/explores/subscriptions.explore.lkml]: Thanks for catching this.  It was a typo.  I just fixed it.

**Pull request merged**